pkgs/top-level/all-packages.nix

```diff
@@ -9231,6 +9231,8 @@ with pkgs;
 
   icoutils = callPackage ../tools/graphics/icoutils { };
 
+  ida-free = libsForQt5.callPackage ../development/misc/ida-free { };
```

Comment:
Would it be possible to move the package to `pkgs/by-name/id/ida-free/package.nix` and remove this line? You'd need to prefix all Qt 5 packages with `libsForQt5`, at least.